backend/database/structure_index_only.sql

```diff
@@ -66,6 +72,13 @@ ALTER TABLE go_cross_references ADD INDEX fk_go_reference_uniprot_entries (unipr
 ALTER TABLE ec_cross_references ADD INDEX fk_ec_reference_uniprot_entries (uniprot_entry_id ASC);
 -- ALTER TABLE ec_cross_references ADD INDEX fk_ec_terms_reference_go_terms_idx (ec_number_code ASC);
 
+
+-- -----------------------------------------------------
+-- Table `unipept`.`ec_cross_references`
```

Comment:
```suggestion
-- Table `unipept`.`interpro_cross_references`
```

Reply:
Changed in pcvthien/unipept@07c941a60b9ed22acdafb61e7ab5d1f62e5621a3